src/__experimental__/components/simple-color-picker/__snapshots__/simple-color-picker.spec.js.snap

```diff
@@ -270,105 +284,109 @@ exports[`SimpleColorPicker renders as expected 1`] = `
       className="c4"
     >
       <div
-        checked={false}
         className="c5"
-        color="#00A376"
-        data-component="simple-color"
       >
-        <input
-          aria-checked={false}
-          checked={false}
-          className="c6 c7"
-          data-down={false}
-          data-element="input"
-          data-up={false}
-          id="rId-0"
-          name="test-group"
-          onBlur={[Function]}
-          onChange={[Function]}
-          onClick={[Function]}
-          onFocus={[Function]}
-          onKeyDown={[Function]}
-          onMouseDown={[Function]}
-          role="radio"
-          type="radio"
-          value="#00A376"
-        />
         <div
-          className="c8 c9"
+          checked={false}
+          className="c6"
           color="#00A376"
-        />
-      </div>
-      <div
-        checked={true}
-        className="c5"
-        color="#0073C1"
-        data-component="simple-color"
-      >
-        <input
-          aria-checked={true}
-          checked={true}
-          className="c6 c7"
-          data-down={false}
-          data-element="input"
-          data-up={false}
-          id="rId-1"
-          name="test-group"
-          onBlur={[Function]}
-          onChange={[Function]}
-          onClick={[Function]}
-          onFocus={[Function]}
-          onKeyDown={[Function]}
-          onMouseDown={[Function]}
-          role="radio"
-          type="radio"
-          value="#0073C1"
-        />
+          data-component="simple-color"
+        >
+          <input
+            aria-checked={false}
+            checked={false}
+            className="c7 c8"
+            data-down={false}
+            data-element="input"
+            data-up={false}
+            id="rId-0"
+            name="test-group"
+            onBlur={[Function]}
+            onChange={[Function]}
+            onClick={[Function]}
+            onFocus={[Function]}
+            onKeyDown={[Function]}
+            onMouseDown={[Function]}
+            role="radio"
+            type="radio"
+            value="#00A376"
+          />
+          <div
+            className="c9 c10"
+            color="#00A376"
```

Comment:
`div` shouldn't have a `color` prop. Please can you investigate?

Reply:
It works the same as before my PR - it's happening because `StyledSimpleColor` in `simple-color.component.js` has both `checked` and `color` passed on it.